box.py

```diff
@@ -462,11 +465,11 @@ def __box_config(self):
     def __convert_and_store(self, item, value):
         if item in self._box_config['__converted']:
             return value
-        if isinstance(value, dict) and not isinstance(value, Box):
+        if _israwinstance(value, dict):
```

Comment:
Need to add a `Box` parameter to activate using this new way as discussed in #78